src/views/process-view/change-of-name-view/change-of-name-view.test.tsx

```diff
@@ -125,10 +125,6 @@ describe("changeofname/change-of-name-view", () => {
       />,
       options,
     );
-    await userEvent.click(
-      screen.getByText(locale.views.changeofname.actions.cancelProcess),
-    );
-    expect(setCancel.mock.calls[0][0]).toBe(true);
```

Comment:
Can this be moved to the test where the cancel button is rendered instead of removing it entirely? and add tests for the states that Cancel button should be rendered?